kolibri/plugins/coach/assets/src/constants/index.js

```diff
@@ -63,3 +65,30 @@ export const GroupModals = {
   DELETE_GROUP: 'DELETE_GROUP',
   MOVE_LEARNERS: 'MOVE_LEARNERS',
 };
+
+export const pageNameToModuleMap = {
+  [PageNames.RECENT_CHANNELS]: 'reports',
```

Comment:
I believe we should update the router route-definition objects to take both a `storeModule` and `component` attributes. This would get rid of both `pageNameToModuleMap`s and `pageNameToComponentMap`s.